### PR TITLE
Tx ndi

### PR DESCRIPTION
**Pull request by Trouffman**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

Note for Moderators : This is adding an optional package for MS Teams Work/School that is downloadable from the UI :
image

The install link is "version-specific", which might require a bunch of manual update until this is fixed.

Also this is sorted on the Microsoft manifest folder, not sure if you have specific check / rules in place for this folder specifically.

validates & installed tested : 
Screenshot 2024-04-15 102914

NB2 : this package cannot be uninstalled once installed. (even manually)

Thanks

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-04-15 08:08:38.567 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!00007FFE8B9B3F75: (caller: 00007FFE8B90A720) Exception(1) tid(11b4) 80073D12 A main app package is required to install this optional package.  Install the main package first and try again.
> 2024-04-15 08:08:38.676 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FFE8B9B2D2F: (caller: 00007FFE8B9B432C) Exception(2) tid(11b4) 80073CF1 Package was not found.
> 2024-04-15 08:08:38.676 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(126)\WindowsPackageManager.dll!00007FFE8BB28931: (caller: 00007FFE8B9B5853) LogHr(1) tid(11b4) 80073CF1 Package was not found.
>     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FFE8B9B2D2F: (caller: 00007FFE8B9B432C) Exception(2) tid(11b4) 80073CF1 Package was not found.
> 2024-04-15 08:08:38.694 [CLI ] MSIX installer failed: 2147958034

(Automated response - build 876.)

**Comment by Trouffman** [comment]
Indeed. This package (as an add-on) requires the Microsoft.Teams package.

Not sure how this would be managed.

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 5:50:30 PM: 
> 2024-04-15 17:50:19.204 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!00007FFB5EDE76FF: (caller: 00007FFB5ED297DD) Exception(1) tid(d58) 80073D12 A main app package is required to install this optional package.  Install the main package first and try again.

(Automated response - build 876.)

File: manifests/m/Microsoft/TeamsTxNDI/2024.8.1.14/Microsoft.TeamsTxNDI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.TeamsTxNDI
+PackageVersion: 2024.8.1.14
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.17763.0
+InstallerType: msix
+PackageFamilyName: Microsoft.Teams.TxNdi_8wekyb3d8bbwe
+Installers:
```
Comment (stephengillie) [inline manifests/m/Microsoft/TeamsTxNDI/2024.8.1.14/Microsoft.TeamsTxNDI.installer.yaml:11]:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.Teams
Installers:
```

Comment (Trouffman) [reply manifests/m/Microsoft/TeamsTxNDI/2024.8.1.14/Microsoft.TeamsTxNDI.installer.yaml:11]: Good to know the dependencies are now supported.
Tested on win 10 and it works with the depedant package NOT installed, also work for repeat isntall / update.
image

Thanks for the code review!

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Verify:
image

I don't see an Apps and features Settings page entry for this package. I can't verify it in Teams as I don't have a testing account for it.

**Comment by Trouffman** [comment]
> Verify: image
> 
> I don't see an Apps and features Settings page entry for this package. I can't verify it in Teams as I don't have a testing account for it.

This is because this is an add-on to Teams. TO see it you need to go to : 
- Add/remove program
- ... > Advanced Options
- it is at the bottom.
image

I have no idea how to find it programatically.

**Pull request merged**